Telerik.JustMock/Core/MocksRepository.cs

```diff
@@ -466,7 +466,17 @@ internal void DispatchInvocation(Invocation invocation)
                                     ObjectInfo.FromObject(invocation.Instance)
                                     :
                                     ObjectInfo.FromObject(invocation.Method.DeclaringType),
-                            invocation.Args.Select(arg => ObjectInfo.FromObject(arg)).ToArray(),
+                            invocation.Args.Select(
```

Comment:
The whole construction of the InvocationInfo object is hard to read. 
- It is not clear how many arguments are passed to the constructor.
- It is not clear where the calculation of one argument ends and where the next begins.
- Too many nestings.
- In addition to the multiple usages of ternary operators there is a select.
- It will be even harder to debug whether one of the arguments is with the correct value before steping-in the constructor.

Please refactor.

Reply:
@mihail-vladov Correct me if I am wrong, but most of the comments express a subjective opinion related to the style of the code and not to its functionality. My understanding of the main goal of code reviews is to isolate potential defects at an early stage and nothing about the personal taste for coding. The code style issues usually are solved with automated tooling following common sense.

Reply:
@ivo-stoilov The code reviews are what the name implies - a review of the code as a whole. This includes isolation of defects at an early stage but also includes a review of the style of the code. Automated style code can find a lot of issues with the code style but they can't find a bad naming of a assembly/namespace/class/method/variable or code clutter.

Reply:
@mihail-vladov Fixed with b875e16